corehq/apps/export/views.py

```diff
@@ -153,6 +153,20 @@ def user_can_view_deid_exports(domain, couch_user):
             ))
 
 
+def get_exports(domain, export_ids, export_class, username=None):
+    if export_class in [FormExportInstance, CaseExportInstance]:
+        return [export_class.get(export_id) for export_id in export_ids]
+    elif export_class is SMSExportInstance:
+        if not username:
+            raise Exception("Username needed to ensure permissions")
+        include_metadata = MESSAGE_LOG_METADATA.enabled(username)
+        return [SMSExportInstance._new_from_schema(
+            SMSExportDataSchema.get_latest_export_schema(domain, include_metadata)
+        )]
+    else:
+        raise Exception("Unexpected export cls %s" % export_class)
```

Comment:
I assume you're reasonably certain this won't happen?

Reply:
yup.